src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ProjectConfigurationsInferredFromUsageCapabilitiesProvider.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.ComponentModel.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    [Export(ExportContractNames.Scopes.UnconfiguredProject, typeof(IProjectCapabilitiesProvider))]
+    [AppliesTo(ProjectCapabilities.AlwaysApplicable)]
+    internal sealed class ProjectConfigurationsInferredFromUsageCapabilitiesProvider : UnconfiguredProjectCapabilitiesProviderBase
+    {
+        private static readonly ImmutableHashSet<string> _capabilities = ImmutableHashSet<string>.Empty.Add(ProjectCapabilities.ProjectConfigurationsInferredFromUsage);
```

Comment:
Bear in mind that this will break C++ immediately - I would have done a check of the extension of the configured project - to only include C#/VB.